cranelift/codegen/src/machinst/vcode.rs

```diff
@@ -860,7 +860,7 @@ impl<I: VCodeInst> VCode<I> {
                            disasm: &mut String,
                            buffer: &mut MachBuffer<I>,
                            state: &mut I::State| {
-                if want_disasm {
+                if want_disasm && !inst.is_args() {
```

Comment:
Why hide the synthetic `Args` instructions in the disassembly?

Reply:
I guess it's somewhat subjective, but in the post-regalloc code it seemed like unnecessary clutter as it would always indicate tautologies (`args rdi=rdi, rsi=rsi, rcx=rcx, rdx=rdx, ...`). Happy to remove this special case if others disagree though :-) (To be clear, it still remains in the VCode debug print either way; the `want_disasm` mechanism is separate)

Reply:
What about cases where the args were passed on the stack? Would those end up being useful to see?

Reply:
Ah, those are actually separately visible as loads, since `args` only handles registers. I suppose in the future if/when RA2 has a notion of "custom stackslot locations" we can give vregs constraints that directly indiciate where to load from, then we don't need that distinction; at that time we can do something different here.

Reply:
That sounds good to me, thanks for the clarification!